tests/test_bgp_neighbor_af.rb

```diff
@@ -117,6 +184,16 @@ def test_nbrs_with_masks
     @@matrix.each do |k, v|
       asn, vrf, nbr, af = v
       nbr += (nbr[/:/]) ? '/64' : '/16'
+      if platform == :ios_xr
+        # XR does not do slash masks and neighbors
+        # so make sure to catch the error
+        assert_raises(ArgumentError, 'Neighbors do not support /') do
+          RouterBgpNeighborAF.new(asn, vrf, nbr, af, true)
+        end
+        # Finish the test case with regular neighbor
+        # to avoid skipping whole testcase
```

Comment:
Fair enough. BTW when I add a 'return' rubocop tells me:

Non-local exit from iterator, without return value. `next`, `break`, `Array#find`, `Array#any?`, etc. is preferred. [Lint/NonLocalExitFromIterator] [2 times]

This is probably why I elected to add a dummy address and just let it run through. Any idea what I can use instead? break / next doesn't work in this context.